Sources/Redis/Docs.docc/index.md

```diff
@@ -0,0 +1,7 @@
+# ``Redis``
+
+Redis is a library to integrate [RediStack](https://github.com/Mordil/RediStack) with Vapor. It provides a number of extensions to both make configuring a Vapor application with Redis simple and to provide Redis implementations for Vapor protocols, such as the cache.
```

Comment:
The GitLab repo is the canonical source, let’s link to it instead

Reply:
Ah thought I had - fixed!